### DISCUSSION TITLE
useLocalSearchParams type parameters not working

### DISCUSSION THREAD

**Discussion by fukuli053** [expo/router#790] [Q&A]
I have created the [space_id].tsx page and I'm trying to retrieve parameters using useLocalSearchParams, but TypeScript is throwing a type error and in all pages.

<img width="770" alt="[space_id].tsx" src="https://github.com/expo/router/assets/44720481/94e290e0-6101-4c8d-8071-00db237639a4">

**Comment by dannyhw** [comment]
@fukuli053 I'm also running into this, did you find any solution?
